src/pages/[...slug].tsx

```diff
@@ -425,8 +427,33 @@ const ContentPage: NextPage<ContentPageProps> = ({
         </div>
     );
 
+    const og: any = {
+        url: seo?.og_url,
+        title: seo?.og_title,
+        type: seo?.og_type,
+        description: seo?.og_description,
+    };
+
+    og['images'] = seo?.og_image?.url ? [{ url: seo?.og_image?.url }] : [];
+
+    const twitter: any = {
+        handle: seo?.twitter_creator,
+        site: seo?.twitter_site,
+    };
+
     return (
         <>
+            <NextSeo
+                title={`${title} | MongoDB`}
```

Comment:
is this Mongodb an ask from elizabeth ?

Reply:
Yes, this is on the metadata epic https://jira.mongodb.org/browse/DEVHUB-1123